src/stactools/palsar2_scansar/stac.py

```diff
@@ -30,88 +38,147 @@ def create_collection() -> Collection:
+    sar.instrument_mode = c.SCANSAR_SAR["instrument_mode"]
+    sar.center_frequency = c.SCANSAR_SAR["center_frequency"]
+    sar.resolution_range = c.SCANSAR_SAR["resolution_range"]
+    sar.resolution_azimuth = c.SCANSAR_SAR["resolution_azimuth"]
+    sar.pixel_spacing_range = c.SCANSAR_SAR["pixel_spacing_range"]
+    sar.pixel_spacing_azimuth = c.SCANSAR_SAR["pixel_spacing_azimuth"]
+    sar.looks_equivalent_number = c.SCANSAR_SAR["looks_equivalent_number"]
+
+    assets = ItemAssetsExtension.ext(collection, add_if_missing=True)
+    assets.item_assets = c.SCANSAR_ASSETS
+
     return collection
 
 
-def create_item(asset_href: str) -> Item:
+def create_item(
+    granule_href: str,
+    **kwargs: Any,
+) -> Item:
     """Create a STAC Item
 
     This function should include logic to extract all relevant metadata from an
     asset, metadata asset, and/or a constants.py file.
 
     See `Item<https://pystac.readthedocs.io/en/latest/api.html#item>`_.
+    Example:
+        ALOS2397743750-211004_WBDR2.2GUD_summary.xml
 
     Args:
-        asset_href (str): The HREF pointing to an asset associated with the item
+        granule_href (str): The HREF pointing to a granule summary xml
 
     Returns:
         Item: STAC Item object
     """
 
-    properties = {
-        "title": "A dummy STAC Item",
-        "description": "Used for demonstration purposes",
-    }
-
-    demo_geom = {
-        "type": "Polygon",
-        "coordinates": [[[-180, -90], [180, -90], [180, 90], [-180, 90], [-180, -90]]],
-    }
+    metalinks = MetadataLinks(
+        granule_href,
+        **kwargs,
+    )
 
-    # Time must be in UTC
-    demo_time = datetime.now(tz=timezone.utc)
+    granule_id = granule_href.replace("_summary.xml", "")
+    product_metadata = ProductMetadata(granule_href, metalinks.manifest)
+
+    asset_href = f"{granule_id}_HH_SLP.tif"
+    with rasterio.open(asset_href) as dataset:
+        bbox = list(dataset.bounds)
+        # geometry = mapping(box(*bbox))
+        # transform = dataset.transform
+        # shape = dataset.shape
+
+        item = Item(
+            id=os.path.basename(granule_id),
+            properties={},
+            geometry=product_metadata.geometry,
+            bbox=product_metadata.bbox,
+            datetime=product_metadata.get_datetime,
+            stac_extensions=[],
+        )
 
-    item = Item(
-        id="my-item-id",
-        properties=properties,
-        geometry=demo_geom,
-        bbox=[-180, 90, 180, -90],
-        datetime=demo_time,
-        stac_extensions=[],
-    )
+        # It is a good idea to include proj attributes to optimize for libs like stac-vrt
+        proj_attrs = ProjectionExtension.ext(item, add_if_missing=True)
+        proj_attrs.epsg = dataset.crs.to_epsg()
+        proj_attrs.bbox = bbox
+        proj_attrs.shape = product_metadata.get_shape  # Raster shape ProductImageSize
+        # proj_attrs.transform = [-180, 360, 0, 90, 0, 180]  # Raster GeoTransform
```

Comment:
Transform would be good to include, as it's used by (e.g.) **odc-stac** and **stackstac** to load data.

Reply:
oops, just forgot to go back and fix that.